images/Containerfile

```diff
@@ -12,6 +12,7 @@ RUN dnf install -y epel-next-release epel-release
 
 RUN dnf install \
         --allowerasing \
+        /usr/bin/python \
```

Comment:
This looks out of place...?

Reply:
This is there to have the "unversioned" `/usr/bin/python` executable. This is just a symlink to `./python3`. It is not there normally, but is provided by the package `python-unversioned-command`. I install it, so that a plain `python` command also works.

With `dnf install`, it's IMO perfectly fine to specify the file you want to install (`/usr/bin/python`), instead of the package name `python-unversioned-command`. I did it in this case, because it seams clearer what this is.

Reply:
ah i got it, it was weird to see it like this